search/search_tests/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+include_directories(${OMIM_ROOT}/3party/jansson/src)
+
+add_compile_options(
```

Comment:
Why do we need this? These options should come from the parent cmakelist, right?